src/main/scala/com/libertyglobal/odh/kafkatool/aclmanager/ACLManager.scala

```
@@ -0,0 +1,41 @@
+package com.libertyglobal.odh.kafkatool.aclmanager
+
+import java.util.concurrent.{ExecutionException, TimeUnit}
+
+import org.apache.kafka.clients.admin.AdminClient
+import org.apache.kafka.common.acl.{AclBinding, AclBindingFilter}
+
+import scala.collection.JavaConverters._
+
+object ACLManager {
+
+  def add(kafka: AdminClient, acls: Seq[AclBinding], dryRun: Boolean = false): Boolean = {
+    try {
+      if (!dryRun) {
+        kafka.createAcls(acls.asJavaCollection).all().get()
+      }
+      true
+    } catch {
+      case ee: ExecutionException => false
```

please never ever swallow exceptions